randomActivityGen.py

```diff
@@ -10,6 +10,7 @@
 
 Other Options:
     --gates.count N             Number of city gates in the city [default: 4]
+    --schools.count H           Number of schools in the city, if not used, number of schools is based on population [default: population-based]
```

Comment:
Why H? I think we should use N for all integers :D And can I suggest "auto" instead of "population-based". Thoughts?

Reply:
Yep, sounds good. H was used when I didn't quite get the parser, and forgot to change it. auto sounds fine.